.babelrc

```diff
@@ -1,9 +1,10 @@
 {
   "presets": [
     ["env", {
-      "modules": false
+      "modules": "commonjs"
```

Comment:
Don't change it. For browser you don't need to transpile modules, because webpack is able to resolve it. What you have to do is to add "env" section in the babel config, define "test" property and override presets array there with
```
["env", {
  "target": { 
    "node": "current"
  }
}]
```